src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -50,6 +50,11 @@ public enum IndexComponent
      */
     TERMS_DATA("TermsData"),
 
+    /**
+     * Product Quantization store used to store compressed vectors for the vector index
+     */
+    VECTORS("Vectors"),
```

Comment:
The javadoc is correct but IMO the name is misleading since the actual vectors are stored inline with the graph.

COMPRESSED_VECTORS is a bit of a mouthful, PQ_VECTORS is probably more reasonable

Reply:
At one point we had this as `PRODUCT_QUANTIZATION`, but that seemed to be meaningless to anyone. I think `COMPRESSED_VECTORS` is clearest, even though it is a bit long. `PQ_VECTORS` is shorter but isn't clear as to what it contains (unless you know).